### DISCUSSION TITLE
when chia.  create a FREE IPFS system

### DISCUSSION THREAD

**Discussion by programsalon** [Chia-Network/chia-blockchain#7591] [Ideas]
I expect a FREE IPFS system like BT, free upload, free or pay for download, no mortgage when mine ChiaFIL Coin, less network bandwidth or support ADSL share. we can named it ChiaFIL?

**Comment by SlugFiller** [comment]
IPFS is often misunderstood. It is NOT crypto. It's actually closer to older P2P like Kazaa. For content to be on IPFS, someone needs to WILLINGLY seed it. If the seed is gone, the content is gone. No part of the protocol describes or enables an unwilling unaware relay seed. No cryptography is used.

Some services build on top of IPFS, by allow for encrypted unaware seeds. However, to motivate such seeds, they are given rewards that are distributed not by an automated architecture, but by a central reseller. Even if said rewards are distributes as crypto, this does not make it a crypto system.

Since IPFS does not use blockchain, combining it with a blockchain like Chia doesn't make any sort of sense. While Chia does use disk space, the plots MUST have a specific format for mining to be secure, and can't be "colored" or repurposed for other types of storage.

Crypto storage is still not a thing, and Chia isn't going to turn it into a thing.

**Reply by programsalon** [reply]
I mean ChiaFIL project is a new IPFS project, not based on Chia. The IPFS projects like FIL,AR... not practical because they are no free upload, they are too expensive for app users.
